src/shared/scrapers/US/TX/harris-county.js

```diff
@@ -0,0 +1,156 @@
+  };
+};
+
+const scraper = {
+  county: 'fips:48201',
+  state: 'iso2:US-TX',
+  country: 'iso1:US',
+  url:
+    'https://app.powerbi.com/view?r=eyJrIjoiYjU5MzU4NjAtZWJjMC00MTllLTkwYjYtMzE4ODY1YjAyMGU2IiwidCI6ImI3MjgwODdjLTgwZTgtNGQzMS04YjZmLTdlMGUzYmUxMGUwOCIsImMiOjN9',
+  sources: [
+    {
+      name: 'SouthEast Texas Regional Advisory Council',
+      url: 'https://www.setrac.org/',
+      description: 'SouthEast Texas Regional Advisory Council'
+    }
+  ],
+  _counties: ['Harris County'],
+  timeseries: true,
+  aggregate: 'county',
+  type: 'json',
+  maintainers: [maintainers.sglyon],
+  scraper: {
+    '0': async function() {
+      const callback = async page => {
+        page.setDefaultTimeout(5000);
+        // Wait for ICU button to load then navigate to that page
+        (await page.waitForXPath("//span[text()='Hospital/COVID Census']/..")).click();
+
+        // Now wait for harris county selector to load
+        (await page.waitForXPath("//div[@aria-label='Harris']")).click();
+
+        /* PowerBI dynamicly swaps out rows so we
+     can't get it all without some scrolling magic
+  */
+        // fill start date  back to beginning
+        // const startDate = await page.waitForXPath("//div[@class='date-slicer']//input[contains(@aria-label, 'Start')]")
+        // await startDate.click({clickCount: 3})
+        // await page.keyboard.type("3/18/2020")
+        // await page.waitFor(1000)
+        // await startDate.press("Enter")
+
+        // use context menu to open table version of chart
+        await (await page.$$('div.visual-lineChart'))[0].click({ button: 'right' });
+        const tableButton = await page.waitForXPath("//drop-down-list-item[//h6[text()='Show as a table']]", {
+          visible: true
+        });
+
+        await page.waitFor(2000);
+        await tableButton.click();
+
+        // Get the data
+        const data = await page.waitForXPath("//div[@aria-label='Grid']").then(getDataFromPivotTable);
+
+        // return now as we don't need the browser anymore
+        return data;
+      };
+      // fetch raw data
+      const data = await fetch.fetchHeadlessCallback(this, this.url, callback);
+
+      // Now parse it out
+      const dates = data.dates
+        .map(x => new Date(x))
+        .map(datetime.parse)
+        .map(x => x.toString());
+      const nRows = dates.length;
+      const nCols = data.colnames.length;
+
+      if (nRows * nCols !== data.data.length) {
+        throw new Error('Number of data entries does not match nrows * ncols');
+      }
+      const hospitalCol = data.colnames.indexOf('Patients in General Beds (Suspected + Confirmed)');
+      const icuCol = data.colnames.indexOf('Patients in Intensive Care Beds (Suspected + Confirmed)');
+
+      const outData = {};
+      for (let ix = 0; ix < nRows; ix++) {
+        outData[dates[ix]] = {
+          hospitalized_current: hospitalCol >= 0 ? data.data[hospitalCol * nRows + ix] : undefined,
```

Comment:
This line doesn't look right at all, just on reading it.  How is `hospitalCol * nRows + ix` getting the right data?  oh, I see, it's how you're loading the table ... that doesn't read well.  ok, working on it.